docs/serverless/03-01-supported-webhooks.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Supported webhooks
+type: Details
+---
+
+A newly created or updated Function CR is first updated by the defaulting webhook and then verified by the validation webhook before the Function Controller starts to process it:
```

Comment:
Same as above, maybe A newly created or modified Function CR is first updated by the...